bin/local-dev-server-certs/README.md

```diff
@@ -0,0 +1,4 @@
+# Local server SSL certs
+
+These certs are used to enable SSL for local testing of discopane. They could probably be used elsewhere
```

Comment:
This should mention how the certs are specifically for `example.com` because it's not obvious from the encoded cert data.